Library/Formula/metashell.rb

```diff
@@ -17,11 +10,16 @@ class Metashell < Formula
   def install
     ENV.cxx11
     # Build internal Clang
-    mkdir "templight/build" do
+    mkdir "3rd/templight/build" do
       system "cmake", "../llvm", "-DLIBCLANG_BUILD_STATIC=ON", *std_cmake_args
       system "make", "clang"
       system "make", "libclang"
       system "make", "libclang_static"
+      system "make", "templight"
+    end
+
+    if MacOS.version == :yosemite
```

Comment:
That should not be necessary at all. Sorry, the [build script](https://github.com/sabel83/metashell/blob/v2.1.0/build.sh#L86) is a bit out of date.